packages/clarity-js/src/layout/node.ts

```diff
@@ -182,6 +182,16 @@ export default function (node: Node, source: Source): Node {
                     let linkData = { tag, attributes };
                     dom[call](node, parent, linkData, source);
                     break;
+                case "VIDEO":
+                case "AUDIO":
+                case "SOURCE":
+                    // Ignoring any base64 src attribute for media elements to prevent big unused tokens to be sent and shock the network 
+                    if (Constant.Src in attributes && attributes[Constant.Src].startsWith("data:")) {
+                        delete attributes[Constant.Src]
```

Comment:
do mediaTag.attributes[Constant.Src] = empty string instead of deleting here

Reply:
also how is empty src handled on visualiaze?

Reply:
An empty "src" attribute results in displaying an HTML media element, such as audio/video, with a playback length of 0.00.